man/rarun2.1

```diff
@@ -142,9 +142,9 @@ Debugging a program redirecting io to another terminal
   $ tty ; clear ; sleep 999999
   /dev/ttyS010
   ## in another terminal run r2
-  $ r2 \-e dbg.profile=foo.rr2 -d ls
+  $ r2 \-r foo.rr2 -d ls
   ## or you can use -R option
-  $ r2 \-R foo.rr2 -d ls
+  $ r2 \-R stdio=/dev/ttys010 -d ls
```

Comment:
The purpose of the example is to show rarun2 profile

Reply:
What do you mean? This seems right to me. You should use `-r` to load the rr2 profile and `-R` to specify the rarun2 options.

Reply:
Then change the example to `-r foo.rr2` and not `-R stdio=/dev/ttys010`

Reply:
did you read this line https://github.com/radare/radare2/pull/11426/files#diff-a204414826ebadcea754af6991545aacR145 ?

Reply:
@Maijin -r foo.rr2 is also included as an example. The purpose is to show how to connect rarun2 to r2 locally.